### ISSUE TITLE
typescript error: useLenis callback variable

### ISSUE THREAD

**Issue by dana-blackbean** [studio-freight/react-lenis#8]
I am working with typescript on a NextJS build. I am trying to replace an already in-use implementation where I am using useLenis() to swap out something like the below code:

```
  const lenis = new Lenis({
    duration: 0.75,
    easing: (t: any) => Math.min(1, 1.001 - Math.pow(2, -10 * t)), // https://www.desmos.com/calculator/brs54l4xou
    direction: 'vertical', // vertical, horizontal
    gestureDirection: 'vertical', // vertical, horizontal, both
    smooth: true,
    mouseMultiplier: 0.75,
    smoothTouch: false,
    touchMultiplier: 4,
    infinite: false,
  })

// SWAP WITH

  const lenis = useLenis();
```

_Note: Lenis '@studio-freight/react-lenis' is already implemented in the base template._

```
  <Lenis root options={{
    duration: 0.75,
    easing: (t: any) => Math.min(1, 1.001 - Math.pow(2, -10 * t)), // https://www.desmos.com/calculator/brs54l4xou
    orientation: 'vertical', // vertical, horizontal
    gestureOrientation: 'vertical', // vertical, horizontal, both
    smoothWheel: true,
    wheelMultiplier: 1.5,
    smoothTouch: false,
    touchMultiplier: 4,
    infinite: false,
    }}>
  </Lenis>
```

When using useLenis() you get the error:

Expected 1-3 arguments, but got 0. ts(2554)
index.d.ts(1, 26): An argument for 'callback' was not provided.

A possible fix could be to "callback" variable optional.

```
export function useLenis(callback: any, deps?: any[], priority?: number): any;
export const ReactLenis: any;
export { ReactLenis as Lenis };
```

I am also seeing that some default lenis tools like scrollTo do not seem to be working with useLenis(). I am not sure if my implementation is incorrect or if this has something to do with the callback as well.

`lenis.scrollTo("#insights", { immediate: true })`

Any help pointing me in the correct direction would be appreciated!

**Comment by clementroche** [comment]
types are fixed on 0.0.17

are you calling `lenis.scrollTo("#insights", { immediate: true })` in a useEffect ? If so you need to add lenis as deps

**Comment by dana-blackbean** [comment]
Great! Yea, I had lenis set up as a deps. I will give it another shot with this update when I have some time! I will mark this as resolved for the moment.
